docs/documentation/securing_apps/topics/oidc/javascript-adapter.adoc

```diff
@@ -211,16 +161,15 @@ For the Hybrid flow, you need to pass the parameter `flow` with value `hybrid` t
 ----
 keycloak.init({
     flow: 'hybrid'
-})
+});
 ----
 
 [#hybrid-apps-with-cordova]
 ==== Hybrid Apps with Cordova
 
-Keycloak support hybrid mobile apps developed with https://cordova.apache.org/[Apache Cordova]. The JavaScript adapter has two modes for this: `cordova` and `cordova-native`:
+Keycloak support hybrid mobile apps developed with https://cordova.apache.org/[Apache Cordova]. The adapter has two modes for this: `cordova` and `cordova-native`:
 
-The default is cordova, which the adapter will automatically select if no adapter type has been configured and window.cordova is present.
-When logging in, it will open an https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-inappbrowser/[InApp Browser] that lets the user interact with {project_name} and afterwards returns to the app by redirecting to `http://localhost`. Because of this, you must whitelist this URL as a valid redirect-uri in the client configuration section of the Admin Console.
+The default is `cordova`, which the adapter will automatically select if no adapter type has been explicitly configured and `window.cordova` is present. When logging in, it will open an https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-inappbrowser/[InApp Browser] that lets the user interact with {project_name} and afterwards returns to the app by redirecting to `http://localhost`. Because of this, you must whitelist this URL as a valid redirect-uri in the client configuration section of the Admin Console.
```

Comment:
```suggestion
The default is `cordova`, which the adapter automatically selects if no adapter type has been explicitly configured and `window.cordova` is present. When logging in, it opens an https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-inappbrowser/[InApp Browser] that lets the user interact with {project_name} and afterwards returns to the app by redirecting to `http://localhost`. Because of this behavior, you whitelist this URL as a valid redirect-uri in the client configuration section of the Admin Console.
```